bin2tif/readme.md

```diff
@@ -42,6 +42,69 @@ docker run \
   terra-ext-bin2tif
 ```
 
+### TORQUE/PBS
+The extractor can also be run on ROGER via the TORQUE/PBS batch system.
+
+First, create a Python virtualenv with the relevant dependencies:
+
+```
+module load gdal2-stack
+module load python/2.7.10
+module load pythonlibs/2.7.10
+virtualenv terraref --python=python2.7
+cd terraref
+. bin/activate
+git clone https://opensource.ncsa.illinois.edu/stash/scm/cats/pyclowder2.git
+pip install  pyclowder2/
+git clone https://github.com/terraref/terrautils/
+pip install terrautils/
+git clone https://github.com/terraref/extractors-stereo-rgb
+pip install Pillow enum34 pyyaml pika functools32 pyparsing pytz GDAL
```

Comment:
we should also be able to do pip install -r requirements.txt to install these dependencies for terrautils/pyclowder2, that will at least cover Pillow, pika, enum, etc.

Reply:
Actually, your existing environment in /projects/arpae/terraref/shared/extractors makes more sense to me than how I've done it.